frontend/packages/integration-tests-cypress/support/resources.ts

```diff
@@ -0,0 +1,61 @@
+import { plural } from 'pluralize';
+
+import { K8sResourceKindReference } from '@console/internal/module/k8s';
+
+export {};
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      resourceShouldBeDeleted(
+        namespaced: boolean,
+        namespace: string,
+        resource: string,
+        name: string,
+      ): Chainable<Element>;
+    }
+  }
+}
+
+// Convert types in the `user.openshift.io~v1~Group` format to `groups.v1.useropenshift.io`
+// to pass to oc.
+const toCLIType = (type: K8sResourceKindReference | string): string => {
+  if (!type.includes('~')) {
+    return type;
+  }
+  const [group, version, kind] = type.split('~');
+  // Resources aren't required to follow this pattern when converting from kind to plural,
+  // but this should work for most resources and is good enough for our tests.
+  return `${plural(kind.toLowerCase())}.${version}.${group}`;
+};
+
+// any command added below, must be added to global Cypress interface above
+
+Cypress.Commands.add(
+  'resourceShouldBeDeleted',
+  (namespaced: boolean, namespace: string, resource: string, name: string) =>
+    cy
+      .exec(
+        `oc get ${namespaced ? `-n ${namespace}` : ''} ${toCLIType(
+          resource,
+        )}/${name} -o template --template '{{.metadata.deletionTimestamp}}'`,
+        { failOnNonZeroExit: false },
+      )
+      .then((result) => {
+        if (result.code === 1) {
```

Comment:
Don't check specifically for `1`. Check if the exit code is non-zero.

Reply:
fixed.  Latest code is in this [commit](https://github.com/openshift/console/pull/4831/commits/afeadc07dd2ee8762bc3053b9158a0cd65221bb8).